inspirehep/modules/theme/static/scss/holding-pen/holding-pen.scss

```diff
@@ -190,7 +190,7 @@ $hp_table_bg: #fff;
       margin-top: -8px;
       border-radius: 3px;
       font-size: .9em;
-
+      color: #696565;
```

Comment:
could we use ``$grey``, WDYT? Just for reusability :)

Reply:
Just updated the PR with this change :)